xclim/indices/_anuclim.py

```diff
@@ -403,6 +403,7 @@ def prcptot_warmcold_quarter(
     return out
 
 
+# FIXME: `src_timestep` is not used here.
```

Comment:
These two FIXME could also be fixed here. But if removing the arg creates bugs in the tests, we can wait another PR.

Reply:
Alright, let's keep the FIXME's there and wait for another PR.